values.yaml

```diff
@@ -79,22 +127,27 @@ solr:
   enabled: true
   host: localhost
   port: 8983
+  storage: 8Gi
+  storageAccessMode: ReadWriteOnce
 
 # To use ingress for routing set ingress.enabled value to true and istio.enabled value to false
 ingress:
   enabled: true
```

Comment:
I know you didn't touch this, but I am going to suggest changing it to false by default. It makes no sense to have it enabled by default anyway, you need to configure the rest of the settings for it to work.